.github/workflows/main.yaml

```diff
@@ -89,6 +89,8 @@ jobs:
             request-scan: "true"
           - dir: "./grafana"
             container-image: "grafana"
+          - dir: "./grafana_plugins_init"
+            container-image: "grafana_plugins_init"
```

Comment:
Also, I think grafana_plugins_init should be scanned as grafana is.

Reply:
Same as the previous one:
ref: https://github.com/cybozu/neco-containers/actions/runs/7828640467/job/21358963112#step:4:178